core/src/main/scala/tech/beshu/ror/accesscontrol/blocks/variables/runtime/VariableContext.scala

```diff
@@ -70,6 +70,7 @@ object VariableContext {
     implicit val externalAuthenticationRule: VariableUsage[ExternalAuthenticationRule] = NotUsingVariable
     implicit val externalAuthorizationRule: VariableUsage[ExternalAuthorizationRule] = NotUsingVariable
     implicit val fieldsRule: VariableUsage[FieldsRule] = NotUsingVariable
+    implicit val responseFieldsRule: VariableUsage[ResponseFieldsRule] = NotUsingVariable
```

Comment:
the new rule does use variables. I think there is a bug in `fieldsRule` above, so you can fix it (but I advise to do it at the end of the task or test on develop, because it can show us some bugs which were missed.